package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "agendash",
-  "version": "0.3.2",
+  "version": "0.3.3",
```

Comment:
You can use NPM [version](https://docs.npmjs.com/cli/version) command to do this when you're pushing new version to NPM.